pkg/providers/gcp/config.go

```diff
@@ -0,0 +1,59 @@
+package gcp
```

Comment:
@cecobask @laurafitzgerald  Could I ask , maybe we could have similar code/structure (as providers/gcp) for two other cases - AWS and Openshift (providers/aws, providers/openshift). Maybe not for this PR, but have follow-on task

Reply:
The structure is mostly the same with all 3 providers. Not sure I follow what changes you are suggesting to do.
We can sync when you're back from PTO 👍🏻

Reply:
@cecobask , yes, sorry, it looks good